sound/soc/intel/boards/sof_sdw.c

```diff
@@ -909,6 +909,7 @@ static int mc_probe(struct platform_device *pdev)
 	struct snd_soc_card *card = &card_sof_sdw;
 	struct snd_soc_acpi_mach *mach;
 	struct mc_private *ctx;
+	int amp_num = 0, i;
```

Comment:
suggested edits to commit message

ASoC: Intel: sof_sdw: add amp number in components string for ucm

The number of speaker amplifiers may vary between platforms. Ucm
needs to check amp number to include different configuration files.
This patch keeps track of the number of speaker amplifiers and stores it in components string of the card.